src/openqaoa-core/openqaoa/algorithms/workflow_properties.py

```diff
@@ -264,15 +264,34 @@ def __init__(
     #             f"cvar_alpha must be between 0 and 1. Received {value}.")
     #     self._cvar_alpha = value
 
-
 class ErrorMitigationProperties(WorkflowProperties):
     """
-    Optional, choose an error mitigation technique for the QAOA circuit. Currently supports only SPAM Twirling.
+    Optional, choose an error mitigation technique for the QAOA circuit.
+
+    Parameters
+    ----------
+    error_mitigation_technique: str
+        The name of the error mitigation technique.
+    """
+    
+    def __init__(
+        self,
+        error_mitigation_technique: Optional[str] = None,
+    ):
+        self.error_mitigation_technique = (
+            error_mitigation_technique.lower()
+            if type(error_mitigation_technique) == str
+            else error_mitigation_technique
+        )
+
+class Spam_Twirling_ErrorMitigationProperties(ErrorMitigationProperties):
+    """
+    choose Spam Twirling as your error mitigation technique for the QAOA circuit.
 
     Parameters
     ----------
     error_mitigation_technique: str
-        The name of the error mitigation technique. Only 'spam_twirling' supported for now.
+        The name of the error mitigation technique.
```

Comment:
Add the `str` that triggers creation of a `Spam_Twirling_ErrorMitigationProperties` object.

Reply:
I don't understand what modifications to perform. Please verify if the last commits addressed this issue as well